### DISCUSSION TITLE
Getting started - how?

### DISCUSSION THREAD

**Discussion by PureKrome** [NetTopologySuite/NetTopologySuite#462] [Ideas]
Hi NTS team,

### TL;DR;

There's no documentation to help anyone get started with NTS. If there is, it's really really _really_ hard to find.
The [API docs](http://nettopologysuite.github.io/NetTopologySuite/api/NetTopologySuite.html) are too low level and don't explain _how to start_. They are just the xml code docs, _I'm assuming_.

### Detailed Question

a number of times, over the years, I've tried to learn how I can do geospatial stuff on .NET with NTS. I've learnt some basic geospatial skills by using MS-Sql server, so I have:

- a number of scenarios
- some basic understanding how to do this, using _their_ "API" which i think is based off OGC. (e.g. [STIntersects](https://docs.microsoft.com/en-us/sql/t-sql/spatial-geography/stintersects-geography-data-type?view=sql-server-ver15), [STGeomFromText](https://docs.microsoft.com/en-us/sql/t-sql/spatial-geography/stgeomfromtext-geography-data-type?view=sql-server-ver15), etc)

Now, trying to break free of MS-Sql Geography/Spatial Types .. and avoiding using EF/EF-Core .. I've tried to learn how to leverage NTS to "do the equivalent spatial stuff" inmemory/in my code. Not having to hit Sql Server or some other geospatial enabled db.

**But ... I just can't get started**. Literally.

I've tried to look around for some documentation, wiki, tutorials .. _anything_ which might at least kick start me .. and then I might bumble along finding various methods which can help me do stuff.

But there's nothing. Nada. Ziltch.

I don't understand why this is 😢 

I totally understand that documentation is a crappy part of OSS. I _get it_.

But could we have a simple 10 line code example or 2, in the README which would at least kick start all of us.

Start with a simple example or 2, like:

- Point in Poly: given a list of USA State (Polygons) tell me if this lat/long (e.g. gps location) intersects one of them. Which one?
- Load a shapefile
- Join to polygons together. Maybe both pieces of data were shapefiles or geojson files?

Please can we have some small, quick starting docs.

Thank you 🍰

**Comment by FObermaier** [comment]
Here you go:
https://github.com/NetTopologySuite/NetTopologySuite/wiki/GettingStarted

**Reply by PureKrome** [reply]
@FObermaier wow - that's a LOVELY getting started document. covers basically everything I've wanted to do:

- reading in WKT's or WKB's (reading/writing)
- spatial operations (predicates, like Intersects, etc)

and of course, the Factory stuff which I had no idea about but can see how important that is when creating geometries on the fly. Nice!

thank you kindly for getting this information up and into the public. So helpful!

Suggestion: add a link / comment in the README in case people might not know to click on the WIKI.
